test/test.do

```diff
@@ -323,7 +323,7 @@ ipacheckskip ${variable6}, ///
     saving("check06_test03_out.xlsx") ///
     sheetreplace
 	
-	* Test 4
+	/* Test 4

```

Comment:
Huh, yeah I see that now. Ok well then let's leave it commented out for now. Thanks Kelsey!